src/core/display/Container.js

```diff
@@ -544,3 +544,12 @@ Container.prototype.renderCanvas = function (renderer)
         renderer.maskManager.popMask(renderer);
     }
 };
+
+/**
+ * Destroys the container
+ *
+ */
+Container.prototype.destroy = function ()
+{
+    this.children = null;
```

Comment:
We should deffo call ```removeChildren``` too. That way we get rid of the parent references from the children.